### ISSUE TITLE
Looking for more details

### ISSUE THREAD

**Issue by KenyonZhao233** [SeokjuLee/QuadNet#2]
I successfully ran your program, but the final result was somewhat unsatisfactory. I don't know whether my data set was distinguished incorrectly or hyperparameter was set incorrectly. I hope I can get some help from you.

Data set I used GTSRB, the ratio of the training set to test set was set to 7: 3 and 8: 2, whether the seen and unseen list  followed your original setting.

Batchsize：100, the learning rate is the default, and the dimension is 100.

But my result:
Seen acc is generally 98-99%, unseen performs well at 8% and performs poorly at 4%

I sincerely hope to get your help.
